app-editors/vscodium/autobuild/build

```diff
@@ -43,7 +43,7 @@ popd
 tar -czf "$SRCDIR"/vscode.tar.gz -T /dev/null
 
 abinfo "Packaging ..."
-"$SRCDIR"/package_linux_bin.sh
+"$SRCDIR"/dev/build.sh
```

Comment:
According to https://github.com/VSCodium/vscodium/blob/c773d683f46a81a6cdb020ded45d95503a9ac934/.github/workflows/stable-linux.yml#L235
This should be `"$SRCDIR"/build/linux/package_bin.sh`

Reply:
Got it, will fix.

Reply:
I have pushed the commit

Reply:
Thanks!